src/pages/team-page.html

```diff
@@ -214,9 +213,9 @@ <h3 class="team-name" cascaded>[[team.title]]</h3>
           <div class="team-container">
             <template is="dom-repeat" items="[[team.members]]" as="member" on-dom-change="_updateCascadedNodes">
               <div class="member">
-                <paper-material class="card" cascaded>
+                <paper-material class="card"  cascaded>
                   <div class="header">
-                    <div class="photo" style="background-image: url('[[member.photoUrl]]')"></div>
+                    <img class="photo" src="[[member.photoUrl]]"/>
```

Comment:
Oh, I wanted to say I got that from the https://github.com/gdg-x/hoverboard/blob/develop/src/pages/speakers-page.html#L183 but you are right!